src/map/lua/luautils.cpp

```diff
@@ -151,6 +151,19 @@ namespace luautils
         lua_register(LuaHandle, "getAbility", luautils::getAbility);
         lua_register(LuaHandle, "getSpell", luautils::getSpell);
 
+        lua_register(LuaHandle, "CampaignHasBattle", luautils::CampaignHasBattle);
```

Comment:
should these all be globals? I would imagine that any that use a zone as a parameter could be in lua_zone instead?

Reply:
They shouldn't be globals. I wasn't sure where they'd end up, so I wrapped them in campaign.lua until a better solution arose. I'll look into moving them into lua_zone.